Gems/AWSMetrics/Code/Tests/MetricsManagerTest.cpp

```diff
@@ -135,7 +135,7 @@ namespace AWSMetrics
     {
     public:
         // Size for each test metrics event will be 180 bytes.
-        static constexpr int TestMetricsEventSizeInBytes = 180;
+        static constexpr int TestMetricsEventSizeInBytes = 178;
```

Comment:
Why 178?

Reply:
I talked to @junbo75 about this and it comes from the event names and values.  I'll update the comments to provide more info because it confused me too.  Basically, the application_id was using the engine version and I shortened the engine version string from "0.1.0.0" to "1.0.0" which caused fewer bytes to be sent:
![image](https://user-images.githubusercontent.com/26804013/213036379-0cd29fa0-d0f9-4201-9281-0998d31d64a0.png)

Reply:
The clarification is appreciated.